global.json

```diff
@@ -1,5 +1,5 @@
 {
   "sdk": {
-    "version": "3.1.200"
+    "version": "3.1.301"
```

Comment:
Also update the `azurepipelines.yml` to make sure the correct version is available on the build server, the builds are failing because of this now. Update this part in `azurepipelines.yml`:
```
- task: UseDotNet@2
  displayName: Install .Net Core 3.1.200 LTS
  inputs:
    packageType: 'sdk'
    version: '3.1.200'
```